packages/kit/src/components/AccountSelector/hooks/useCreateQrWallet.tsx

```
@@ -20,7 +20,7 @@ import {
 } from '@onekeyhq/shared/src/errors';
 import { ETranslations } from '@onekeyhq/shared/src/locale';
 import { appLocale } from '@onekeyhq/shared/src/locale/appLocale';
-import { EOnboardingPages } from '@onekeyhq/shared/src/routes';
+import { EOnboardingPages, EOnboardingPagesV2 } from '@onekeyhq/shared/src/routes';
```

ESLint

[error] Replace `·EOnboardingPages,·EOnboardingPagesV2·` with `⏎··EOnboardingPages,⏎··EOnboardingPagesV2,⏎`

(prettier/prettier)